scripts/build

```diff
@@ -11,8 +11,30 @@ if [ "$(uname)" = "Linux" ]; then
 fi
 LINKFLAGS="-X main.Version=$VERSION"
 LINKFLAGS="-X main.GitCommit=$COMMIT $LINKFLAGS"
-CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator
-if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
+if [ "$CROSS_ARCH" != "true" ]; then
+  CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator
+  if [ "$CROSS" = "true" ]; then
     GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-darwin
     GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-windows
-fi
+  fi
+else
+  if [ "$ARCH" = "amd64" ]; then
+    CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator
+    GOARCH=arm64 CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator-arm64
+    if [ "$CROSS" = "true" ]; then
+      GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-darwin
+      GOARCH=arm64 GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-darwin-arm64
+      GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-windows
+      GOARCH=arm64 GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-windows-arm64
+    fi
+  else
+    CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o "bin/backup-restore-operator$SUFFIX"
+    GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator
+    if [ "$CROSS" = "true" ]; then
+      GOOS=darwin go build -ldflags "$LINKFLAGS" -o "bin/backup-restore-operator-darwin$SUFFIX"
+      GOARCH=amd64 GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-darwin
+      GOOS=windows go build -ldflags "$LINKFLAGS" -o "bin/backup-restore-operator-windows$SUFFIX"
+      GOARCH=amd64 GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-windows
+    fi
+  fi
+fi
```

Comment:
There's a lot of repetition in this code. I rewrote it with an "XARCH" variable (for the other architecture and came up with this):
```

CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator

# Set CROSS to build for other platforms
if [ "$CROSS" = "true" ]; then
  for OS in darwin windows ; do
      GOOS=$OS go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-$OS
  done
fi

# Set CROSS_ARCH to build for other architectures (one only currently)

if [ "$CROSS_ARCH" == "true" ]; then
  case "$ARCH" in
    amd64) XARCH=arm64 ;;
    arm64) XARCH=amd64 ;;
    *) echo "Unsupported ARCH of $ARCH" 1>&2 ; exit 1
  esac
  GOARCH=$XARCH CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator-$XARCH
  if [ "$CROSS" = "true" ]; then
    for OS in darwin windows ; do
      GOARCH=$XARCH GOOS=$OS go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-${OS}-$XARCH
    done
  fi
fi
```